src/datepicker/datepicker.stories.ts

```diff
@@ -150,6 +152,18 @@ storiesOf("Components|Date Picker", module)
 			</ibm-date-picker>
 		`,
 		props: {
+			language: select("Language of the calendar", [
+				"en",
```

Comment:
Is it useful to spread these over new lines?

Reply:
I suppose it looks better in one line 😄